app/controllers/content_items_controller.rb

```diff
@@ -13,10 +12,7 @@ class ContentItemsController < ApplicationController
   rescue_from PresenterBuilder::SpecialRouteReturned, with: :error_notfound
   rescue_from PresenterBuilder::GovernmentReturned, with: :error_notfound
 
-  before_action :set_explore_menu_response
-  after_action :set_slimmer_template
-
-  helper_method :explore_menu_variant, :explore_menu_variant_b?
+  slimmer_template "core_layout"
```

Comment:
The `core_layout` template is the default template in Slimmer - does it need to be set here?

Reply:
👍